### PR TITLE
docs for escrow function

### PR DESCRIPTION
**Pull request by ovidiuolteanu**
description and disclaimers of the XMEX escrow functionality

### CODE REVIEW

File: docs/xexchange-features/escrow.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ 2. Receiver will only be able to redeem the transferred token ONLY after a certain amount of days;
+
+## XMEX move limitations
+ - Sender can only send to a receiver ONCE per 30 days;
+ - Receiver will be able to redeem the tokens after 30 days;
+ - If receiver did not redeemed the tokens from a specific sender, that sender will not be able to send tokens again to the same receiver. Therefore, the receiver has to redeem the tokens first;
+ - Receiver can't accept new transfers from any other address after a sender sends tokens to it. A 30 days cooldown period is applied to receiver after a transfer was initiated towards the receiver.
+
+## Canceling XMEX moving transfers
+We care about the security of users' accounts. That’s why the contract has a transfer cancelling feature available that allows an Admin to cancel malitious transfers that occur out of account thefts, exploits or scams.
```
Comment (ioandanpopa) [inline docs/xexchange-features/escrow.md:28]: malicious

File: docs/xexchange-features/escrow.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+XMEX is special for its transfer locking mechanism, since it is supposed to be bound to the account. But sometimes people have to move for various reasons, so do tokens.
+The XMEX Move function comes as a solution for this precise and usually inconvenient case.
+
+The XMEX Move (also known as Escrow) contract facilitates the transfer of XMEX from one account to another, with limitations in place to overcome scamming and abuse of token transfers that go against the idea of holding XMEX.
+Therefore, it shall be used mindfull of its strict conditions and, most likely, only in extreme situations.
+
+## XMEX Move flow
+
+ 1. Sender selects the batch of XMEX tokens to be transferred; after Move button clicked, sender has to fill in the receiver address and initiate the tokens transfer;
+ 2. Receiver will only be able to redeem the transferred token ONLY after a certain amount of days;
```
Comment (ioandanpopa) [inline docs/xexchange-features/escrow.md:19]: Receiver will be able to redeem the transferred token ONLY after a certain amount of days;

**Review by ioandanpopa — APPROVED** [review]

File: docs/xexchange-features/escrow.md (hunk 2 above)
Comment (ovidiuolteanu) [reply docs/xexchange-features/escrow.md]: fixed

**Review by bogdan-rosianu — APPROVED** [review]
could have also added a note about lkmex move

**Pull request merged**